swapPrimeNo/SwapPrimeNo.java

```diff
@@ -5,41 +5,47 @@
 public class SwapPrimeNo {
     private int temporary;
     private int lastNumber;
-   private boolean flag = false;
+    private boolean flag = false;
 
     /*
     Q3. Given an array of ints, swap the first and last elements in the array. Return the modified array.
      If the last array value is prime no.
      */
     public static void main(String[] args) {
-        SwapPrimeNo swapObject = new SwapPrimeNo();
         int[] inputArray = {1, 2, 3, 7};
         int[] inputArraySecond = {1, 2, 3};
         int[] inputArrayThird = {1, 6, 7, 9, 8};
-        swapObject.swapPrimeNumber(inputArray);
-        swapObject.swapPrimeNumber(inputArraySecond);
-        swapObject.swapPrimeNumber(inputArrayThird);
+        int lastNumber = inputArray[inputArray.length - 1];
+        swapPrimeNumber(lastNumber, 2);
```

Comment:
No need to call this method here as it is already getting called in printResult method